packages/common/src/types/visualizations.ts

```diff
@@ -0,0 +1,31 @@
+export type BarChartConfig = {
```

Comment:
```suggestion
export type BarChartSqlConfig = {
```

Reply:
I figured we might use these in another context than SQL runner, so leaving this generic is good. There's nothing about it that is tightly coupled to SQL. Thoughts?

Reply:
I called mine `TableChartSqlConfig` but I don't have a strong opinion so happy to change mine. You can keep this one sure 👍